### DISCUSSION TITLE
README Revamp!

### DISCUSSION THREAD

**Discussion by 3kh0** [3kh0/ext-remover#807] [Announcements]
Heyo gamers!

I spent the day redoing the entire README; this makes it a little better to use. Some of the exploits had bad descriptions, so I did some research to make them better and adequately formatted. I hope y'all like it. I also removed some old exploits that do not help. But if you want them back, open a pull request!

Have fun :D

**Comment by AshtonDavies** [comment]
Bro's against me and got rid of LoMoH and LTMEAT Flood. LoMoH is patched in v111 and up, so you can't assume an ancient, yet longer-lasting exploit than LTBEEF is no use. LTMEAT Flood is better than Temp.

**Reply by 3kh0** [reply]
I do not think LoMoH is working, the Google form says you need permission, I am not sure if it works, but I will add them back tomorrow, sorry for the oversight

**Reply by AshtonDavies** [reply]
That's what the create link button is for, because you can't share a quiz outside your organization when locked mode is enabled. And the rickroll is just for patched users fyi.

**Reply by AshtonDavies** [reply]
Temp is not only more difficult, but it's temporary and requires tabs to stay open.

**Comment by Xty21223** [comment]
Awesome work, Echo! I did not see the extra branch tho.
